src/components/AR/frameworks/trackingMode.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '@wonderlandengine/api';
+
+export interface ITrackingMode {
```

Comment:
@Squareys We haven't defined in our coding style how we name interface. So far I haven't used `ISomething`, just used `Something`. Let me know what you prefer.